dali/operators/generic/one_hot.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/generic/one_hot.h"
+
+namespace dali {
+
+void OneHot::RunImpl(HostWorkspace &ws) {
```

Comment:
I have a general question about this: should all new operators be written in operator/kernel API now or is it ok to use still the old method, if appropriate? @mzient @JanuszL

Reply:
This one is so simple so probably there is no point writing a separate CPU kernel for that.